R/spatvec.R

```diff
@@ -48,7 +48,7 @@ setMethod("geomtype", signature(x="Spatial"),
 )
 
 setMethod("geom", signature(x="SpatVector"),
-	function(x, wkt=FALSE, hex=FALSE, df=FALSE, list=FALSE, xnm="x", ynm="y"){
+	function(x, wkt=FALSE, hex=FALSE, df=FALSE, list=FALSE, xnm="x", ynm="y", wkb=FALSE){
```

Comment:
Maybe it would be good idea to move the `wkb=FALSE` before the `df=FALSE`?

Reply:
Done that.